crates/uv-cli/src/lib.rs

```diff
@@ -4672,6 +4672,15 @@ pub struct PythonPinArgs {
     /// `requires-python` constraint.
     #[arg(long, alias = "no-workspace")]
     pub no_project: bool,
+
+    /// Pin the global (user-level) Python version.
+    ///
+    /// This causes uv to write the specified version to a user-level
+    /// `.python-version` file. This will be used as a fallback for any
+    /// project that doesn't contain a `.python-version` file in its
+    /// local directory or ancestor directories.
```

Comment:
Can you adjust this to match the wording and style of the top-level pin description?

https://github.com/astral-sh/uv/blob/7340ff72daca2e837b3383dcd861e63f9c77a477/crates/uv-cli/src/lib.rs#L4425-L4433

I might say something like..

> Update the global Python version pin.
>
> Writes the pinned Python version to a `.python-version` file in the `XDG_CONFIG_HOME` instead of the working directory.
>
> When a local Python version pin is not found in the working directory or an ancestor directory, this version will be used instead.
>
> Unlike local version pins, this version is used as the default for commands that mutate global state, like `uv tool install`.

The line wrap here also doesn't match the rest of the docs.

Reply:
(my doc might need an update to account for platform differences)

Reply:
Updated